src/object_store/src/object/opendal_engine/azblob.rs

```diff
@@ -35,7 +38,14 @@ impl OpendalObjectStore {
         builder.endpoint(&endpoint);
         builder.account_name(&account_name);
         builder.account_key(&account_key);
-        let op: Operator = Operator::new(builder)?.finish();
+        let op: Operator = Operator::new(builder)?
+            .layer(
+                RetryLayer::new()
+                    .with_factor(2.0)
+                    .with_min_delay(Duration::from_secs(1))
+                    .with_max_times(4),
+            )
+            .finish();
```

Comment:
Since all opendal engines use the same RetryLayer config, can we have a common function to create this layer so that we won't miss to change some engines if we need to change the retry config in the future.

Reply:
Changed, and just leave the retry parameters config by specific services to prevent changing retry config for one service.